admin/tools/provision/README.rst

```diff
@@ -0,0 +1,57 @@
+**************************************
+Provision Qserv using Docker+Openstack
+**************************************
+
+Pre-requisite
+-------------
+
+* Install system dependencies
+
+.. code-block:: bash
+
+   sudo apt-get install python-dev python-pip
+
+   # Install the OpenStack client
+   sudo pip install python-openstackclient
+
+   # Install the nova client
+   sudo pip install python-novaclient
+
+* Download Openstack RC file: http://docs.openstack.org/user-guide/common/cli_set_environment_variables_using_openstack_rc.html
+
+* Install shmux to run multi node tests
+see http://web.taranis.org/shmux/
```

Comment:
This line should probably be a part of the list item (previous line, separated by colon)?